projects/ad9361/src/common.h

```diff
@@ -42,19 +42,13 @@
 /******************************************************************************/
 /***************************** Include Files **********************************/
 /******************************************************************************/
+
 #include <stdint.h>
+#include <errno.h>
 
 /******************************************************************************/
 /********************** Macros and Constants Definitions **********************/
 /******************************************************************************/
```

Comment:
nitpick: you can also remove the header `Macros and Constants Definition`

Reply:
Done